scripts/cpu-setonix.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash --login
+
+# SLURM directives
+#
+# Here we specify to SLURM we want an OpenMP job with 32 threads
+# a wall-clock time limit of one hour.
+#
+# Replace [your-project] with the appropriate project name
+# following --account (e.g., --account=project123).
+
+#SBATCH --account=pawsey0807
+#SBATCH --partition=work
+#SBATCH --nodes=2
+#SBATCH --ntasks-per-node=128
+#SBATCH --cpus-per-task=1
+#SBATCH --mem=58880M      #Indicate the amount of memory per node when asking for share resources
+#SBATCH --time=01:00:00
+
+# ---
+# Load here the needed modules
+
+# ---
+# OpenMP settings
```

Comment:
We don't use OpenMP, so we should get rid of all of the OMP stuff to avoid confusion.